src/partials/footer-content.hbs

```diff
@@ -11,6 +11,7 @@
             fill="currentColor" />
         </svg>
       </a>
+<!--
```

Comment:
why not just delete instead of commenting out?

Reply:
Because we'll probably add in again at GH discussion with a different icon? And I can just slot in the new icon file.